src/shared/ui/Accordion/Accordion.tsx

```
@@ -1,70 +1,152 @@
+          >
+            <Icon name="arrowDown" size={20} />
+          </motion.div>
+        )}
+      </button>
+
+      <AnimatePresence initial={false}>
+        {isOpen && (
+          <motion.div
+            id={contentId}
+            aria-labelledby={triggerId}
+            role="region"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<section>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)